.github/workflows/ci.yml

```diff
@@ -0,0 +1,104 @@
+name: infoware CI
+on:
+  push:
+  pull_request:
+
+jobs:
+  linux-x64:
```

Comment:
amd64 (likewise everywhere below) (well, it seems that none of the other "jobs" have the arch listed, so maybe drop it here entirely)

Reply:
Fixed. Switched to `linux:`